windows/setup.ps1

```diff
@@ -128,7 +128,7 @@ Set-Attr $result.ansible_facts "ansible_user_uid" ([int] $user.User.Value.Substr
 Set-Attr $result.ansible_facts "ansible_user_sid" $user.User.Value
 
 # Use English locale
-[System.Threading.Thread]::CurrentThread.CurrentCulture = [System.Globalization.CultureInfo]::GetCultureInfo('en-US')
+$culture = New-Object System.Globalization.CultureInfo('en-US')
```

Comment:
I can't imagine a Windows system without en-US, but  [System.Globalization.CultureInfo]::InvariantCulture is the documented right thing to use here (and saves us an object creation, wee)...